plugin/grpc/README.md

```diff
@@ -129,7 +129,15 @@ Or with multiple upstreams from the same provider
 }
 ~~~
 
+For local upstreams, can using `UNIX Socket` protocol.
+
+~~~ corefile
+. {
+    grpc . path/to/grpc.sock
```

Comment:
I think this example would fail to parse correctly. shouldn't it be `grpc . unix:///path/to/grpc.sock`?

Reply:
No, you're wrong

Grpc is an outer application protocol. The specific communication protocol is unix. They are not a concept.

Reply:
No, you're wrong

Grpc is an outer application protocol. The specific communication protocol is unix. They are not a concept

Reply:
I think you're not understanding my comment.  Let me rephrase...

Will your example parse without error?  In all of your unit test cases, the sock path includes the unix uri prefix.  If this example is valid, please add this style of config to the unit tests.

Reply:
I understand your maind. What I want to express is that what you said is not accurate.
The  `unix://` prefix is necessary, otherwise the resolution will conflict, and the `resolve.conf` definition will be found by default.

Reply:
> The unix:// prefix is necessary

If it is necessary, your example will fail to work as described.  It would assume the path is to a resolv.conf file, parse it as such, and thus be an invalid example of how to use a unix socket.